deployment/src/main/java/io/quarkiverse/jasperreports/deployment/JasperReportsProcessor.java

```diff
@@ -98,22 +99,28 @@ void runtimeInitializedClasses(BuildProducer<RuntimeInitializedPackageBuildItem>
     }
 
     @BuildStep
-    void substrateResourceBuildItems(BuildProducer<NativeImageResourceBuildItem> nativeImageResourceProducer,
+    void registerResourceBuildItems(BuildProducer<NativeImageResourceBuildItem> nativeImageResourceProducer,
             BuildProducer<NativeImageResourceBundleBuildItem> resourceBundleBuildItem) {
         nativeImageResourceProducer.produce(new NativeImageResourceBuildItem(
                 EXTENSIONS_FILE,
                 "default.jasperreports.properties",
                 "jasperreports_messages.properties",
                 "metadata_messages.properties",
                 "metadata_messages-defaults.properties",
```

Comment:
Are these still required, given they are also registered as `NativeImageResourceBundleBuildItem` below?

Reply:
Nevermind, I see they were removed in this PR